lib/dep-graph-builders/npm-lock-v2/index.ts

```diff
@@ -418,8 +418,12 @@ const checkOverrides = (
       continue;
     }
 
-    // If we find a string as override we know we found what we want
-    if (typeof override === 'string') {
+    // If we find a string as override we know we found what we want *if*
+    // we are at our root
+    if (
+      idx + 1 === ancestryWithoutRoot.length &&
+      typeof override === 'string'
+    ) {
```

Comment:
tests?

Reply:
Looking to create some fixtures now to illustrate this behaviour

Reply:
Done